src/EditorFeatures/CSharpTest/Completion/CompletionProviders/OverrideCompletionProviderTests_ExpressionBody.cs

```diff
@@ -99,7 +98,7 @@ class C : B
     public virtual int A() => 2;
     class C : B
     {
-        public override int A() => base.A();$$
+        public override int A() $$=> base.A();
```

Comment:
Out of curiosity, why here?  Why not, say, before 'base'?

Reply:
I'd be fine either way, but recently found myself deleting everything on the right on the position I chose (because I already had a block body on next line). That tipped the scale slightly for me.